pkg/subctl/cmd/show/connections.go

```diff
@@ -46,23 +48,22 @@ func init() {
 		Long:    `This command shows information about submariner endpoint connections with other clusters.`,
 		PreRunE: restConfigProducer.CheckVersionMismatch,
 		Run: func(command *cobra.Command, args []string) {
-			cmd.ExecuteMultiCluster(restConfigProducer, showConnections)
+			execute.OnMultiCluster(restConfigProducer, showConnections)
```

Comment:
You can utilize `IfSubmarinerInstalled` added with https://github.com/submariner-io/submariner-operator/pull/2010.  Same with the other commands.

```suggestion
			execute.OnMultiCluster(restConfigProducer, execute.IfSubmarinerInstalled(showConnections))
```